### ISSUE TITLE
Build-DeploymentPlans function fails when trying to builtin and custom policy sets to as exemptions

### ISSUE THREAD

**Issue by doenoe** [Azure/enterprise-azure-policy-as-code#608]
**Describe the bug**
When trying to add builtin or custom policy sets as exemptions via a .jsonc file as described [here](https://azure.github.io/enterprise-azure-policy-as-code/policy-exemptions/), the `Build-DeploymentPlans` function fails with the following error:

```
Confirm-PolicySetDefinitionUsedExists: C:\Users\user\Documents\PowerShell\Modules\EnterprisePolicyAsCode\10.2.1\internal\functions\Build-ExemptionsPlan.ps1:375:29
Line |
 375 |                              -PolicySetDefinitionsScopes $PacEnvironme …
     |                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | A parameter cannot be found that matches parameter name 'PolicySetDefinitionsScopes'.
```
I tried to debug the error and I found that the `Build-DeploymentPlans` calls the` Build-ExemptionPlans` function, which again calls the `Confirm-PolicySetDefinitionUsedExists` function. The Confirm-PolicySetDefinitionUsedExists function is called with a `-PolicySetDefinitionsScopes` parameter which is not present in the `Confirm-PolicySetDefinitionUsedExists.ps1` file, there is however a parameter present in the file with quite a similar name called `PolicyDefinitionsScopes`.  

**To Reproduce**
Create a .jsonc file and add a custom or builtin policy set to exempt and run the Build-DeploymentPlans function. 

**Expected behavior**
Build-DeploymentPlans should run without errors.

**EPAC Version**
10.2.1 - EnterprisePolicyAsCode   
2.19.0 - Az.Accounts                       
0.13.1 - Az.ResourceGraph                    
6.16.2 - Az.Resources

**Comment by techlake** [comment]
Fixed in v10.2.2

**Comment by doenoe** [comment]
The  `Build-DeploymentPlans` function now fails on another unfound parameter:

```
Confirm-PolicySetDefinitionUsedExists: C:\Users\user\Documents\PowerShell\Modules\EnterprisePolicyAsCode\10.2.2\internal\functions\Build-ExemptionsPlan.ps1:401:29
Line |
 401 |                              -AllDefinitions $AllDefinitions.policyset …
     |                              ~~~~~~~~~~~~~~~
     | A parameter cannot be found that matches parameter name 'AllDefinitions'.
```

After changing the parameters, I got some more errors in other function scripts as well.
